From e711ec8aaddd1da171a645959fbb59c1cae22b11 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Tue, 17 Mar 2020 00:02:14 +0100
Subject: [PATCH] nf-conntrack: improve default values

Conntrack values in OpenWrt are optimized for routers, which has 128 MB
RAM. Let's optimize it for Turris routers, which has at least 512 MB
RAM.

More details about variables and descriptions can be found here:
https://www.kernel.org/doc/Documentation/networking/nf_conntrack-sysctl.txt

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 package/kernel/linux/files/sysctl-nf-conntrack.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/kernel/linux/files/sysctl-nf-conntrack.conf b/package/kernel/linux/files/sysctl-nf-conntrack.conf
index c6a0ef362b..8ce533d985 100644
--- a/package/kernel/linux/files/sysctl-nf-conntrack.conf
+++ b/package/kernel/linux/files/sysctl-nf-conntrack.conf
@@ -3,6 +3,6 @@
 
 net.netfilter.nf_conntrack_acct=1
 net.netfilter.nf_conntrack_checksum=0
-net.netfilter.nf_conntrack_tcp_timeout_established=7440
+net.netfilter.nf_conntrack_tcp_timeout_established=432000
 net.netfilter.nf_conntrack_udp_timeout=60
 net.netfilter.nf_conntrack_udp_timeout_stream=180
-- 
2.35.1
